### PR TITLE
glinux setup script

### PR DESCRIPTION
**Pull request by henry54809**

### CODE REVIEW

File: bin/setup_glinux [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "data-root": "/usr/local/google/docker",
+  "bip": "192.168.9.1/24",
+  "storage-driver": "overlay2",
+  "debug": true,
+  "registry-mirrors": ["https://mirror.gcr.io"]
+}
+EOF
+
+sudo service docker start
+sudo addgroup docker || true
```
Comment (grafnu) [inline bin/setup_glinux]: this part seems generic, and not specific to gLinux -- should it be in the common part of the script?

Comment (henry54809) [reply bin/setup_glinux]: do you mean the addgroup docker part? These are also in the bin/setup_base.

Comment (grafnu) [reply bin/setup_glinux]: Yes -- so why is it in both?

Comment (henry54809) [reply bin/setup_glinux]: I just wanted that script to also be standalone.

Comment (grafnu) [reply bin/setup_glinux]: fair 'nuff

**Pull request merged**